src/routes/scope-selection/ScopeSelectionScreen.tsx

```diff
@@ -48,6 +69,13 @@ const ScopeSelectionScreen = ({ navigation, route }: IntroScreenProps): JSX.Elem
   const { initialSelection } = route.params
   const { data: disciplines, error, loading, refresh } = useLoadDisciplines({ parent: null })
   const { data: selectedProfessions, refresh: refreshSelectedProfessions } = useReadSelectedProfessions()
+  const { data: allProfessions } = useLoadDisciplines({ parent: null, allLevels: true })
+  const [searchString, setSearchString] = useState<string>('')
+  const [showSearchResults, setShowSearchResults] = useState<boolean>(false)
+  const filteredProfessions = useMemo(
+    () => searchProfessions(allProfessions, searchString),
+    [allProfessions, searchString]
+  )
```

Comment:
you should also filter professions the user hes already selected. As it should not be possible to add the same profession twice.